BitwardenShared/UI/Vault/VaultItem/ViewItem/ViewItemProcessor.swift

```diff
@@ -521,7 +520,9 @@ private extension ViewItemProcessor {
                 let hasPremium = await (try? services.vaultRepository.doesActiveAccountHavePremium()) ?? false
                 let hasMasterPassword = try await services.stateService.getUserHasMasterPassword()
                 let collections = try await services.vaultRepository.fetchCollections(includeReadOnly: true)
-
+                let restrictItemDeletionFlagEnabled: Bool = await services.configService.getFeatureFlag(
+                    FeatureFlag.restrictItemDeletion
+                )
```

Comment:
⛏️ I believe you can skip the enum name reference:
```suggestion
                let restrictItemDeletionFlagEnabled: Bool = await services.configService.getFeatureFlag(
                    .restrictItemDeletion
                )
```